### PR TITLE
Adjust Value interface in RootStore by adding QuantityValue type to it

### PR DESCRIPTION
**Pull request by guergana**

### CODE REVIEW

File: src/store/RootState.ts [hunk 1]
```diff
@@ -18,7 +18,13 @@ export interface ItemValue {
 
 export type StringValue = string;
 
-export type Value = ItemValue | StringValue | null;
+export interface QuantityValue {
+	value: number;
+	precision?: number;
+	unit: ItemValue;
```
Comment (micgro42) [inline src/store/RootState.ts]: The unit should either be also optional or it should be nullable.

**Review by micgro42 — CHANGES_REQUESTED** [review]
Almost there! What is missing is the ability for this type to represent a value without unit, e.g. a population 10000.

File: src/store/RootState.ts (hunk 1 above)
Comment (guergana) [reply src/store/RootState.ts]: adressed

**Review by micgro42 — APPROVED** [review]
Looks good, thank you :)

**Pull request merged**